packages/debian/entrypoint.sh

```diff
@@ -45,7 +45,9 @@ buildrepo() {
     cp /distributions /repository/conf
     sed -i 's/_REPONAME_/'"${reponame}"'/g' /repository/conf/distributions
     reprepro -b /repository/ export
-    reprepro -C "$reponame" -b /repository/ includedeb bionic /packages/*.deb
+    if ls /packages/*.deb ; then
```

Comment:
Hope the buildchain won't complain of a missing target file, we will see :crossed_fingers:

Reply:
tested locally seems good